actionpack/test/controller/action_callbacks_test.rb

```diff
@@ -9,13 +9,13 @@ class << self
     end
 
     def before_actions
-      filters = _process_action_callbacks.select { |c| c.kind == :before }
-      filters.map!(&:raw_filter)
+      action_callbacks = _process_action_callbacks.select { |c| c.kind == :before }
+      action_callbacks.map!(&:raw_action_callback)
```

Comment:
Replaced 'filter' with 'action_callback' throughout this file and would like to confirm it reads sensibly.